config/custom-environment-variables.yaml

```diff
@@ -38,6 +38,11 @@ commands:
 
 strategy:
     plugin: STRATEGY
+    disk:
+        # existing dir to store the cache
+        cachePath: DISK_CACHE_PATH
+        # number of milliseconds between each cache cleanup for disk space recovery. Set to 0 to deactivate entirely.
+        cleanEvery: DISK_CLEAN_EVERY
```

Comment:
not sure how clear this variable name is, but since it matches the one used in catbox-disk, I guess it makes sense
I think adding a [link](https://github.com/mirusresearch/catbox-disk#options) in the comment here might be a good idea

Reply:
kk, will add that